src/blog/jspi.md

```diff
@@ -102,17 +102,17 @@ long promiseFib(long x) {
 }
 // promise an addition
 EM_ASYNC_JS(long, promiseAdd, (long x, long y), {
-  return Promise.resolve((k, r) => {
+  return new Promise((resolve, reject) => {
     setTimeout(() => {
-      return k(x + y);
+      return resolve(x + y);
```

Comment:
Not technically wrong, but `return resolve` is weird since `resolve` always returns `undefined`.

Reply:
Done